frame/pablo/src/mock.rs

```diff
@@ -137,6 +158,9 @@ impl pablo::Config for Test {
 	type LbpMaxInitialWeight = MaxInitialWeight;
 	type LbpMinFinalWeight = MinFinalWeight;
 	type PoolCreationOrigin = EnsureRoot<AccountId>;
+	type Time = Timestamp;
+	type EnabledPoolTWAP = IsInVec<EnabledPoolTWAP>;
```

Comment:
This means that we'd need to do a runtime upgrade to add more pools correct? Which means that in order to have a pool with a TWAP, we'd:

1. Add the pool through governance.
2. Do a runtime upgrade.
3. Add the pool to the Pools<T>.

That's too many steps, need to be doable with just extrinsics.